internal/apply/apply.go

```diff
@@ -87,18 +87,24 @@ func applyAll(dir string, projectConfig projectconfig.ZeroProjectConfig, applyEn
 			exit.Fatal("Failed to load module config, credentials cannot be injected properly")
 		}
 
-		// Get project credentials for the makefile
-		credentials := globalconfig.GetProjectCredentials(projectConfig.Name)
-		credentialEnvs := credentials.SelectedVendorsCredentialsAsEnv(modConfig.RequiredCredentials)
-		envList = util.AppendProjectEnvToCmdEnv(mod.Parameters, envList)
-		envList = util.AppendProjectEnvToCmdEnv(credentialEnvs, envList)
+		envVarTranslationMap := modConfig.GetParamEnvVarTranslationMap()
+		envList = util.AppendProjectEnvToCmdEnv(mod.Parameters, envList, envVarTranslationMap)
 		flog.Debugf("Env injected: %#v", envList)
 		flog.Infof("Executing apply command for %s...", modConfig.Name)
 		util.ExecuteCommand(exec.Command("make"), modulePath, envList)
 		return nil
 	})
 }
 
+func getParameterDefinition(modConfig moduleconfig.ModuleConfig, field string) moduleconfig.Parameter {
```

Comment:
Is this function not actually used anywhere?

Reply:
Yeah looks like it, will remove